packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss

```diff
@@ -465,6 +466,10 @@ $Calendar-dayPicker-margin: 10px;
       @include text-align(right);
       top: 199px;
     }
+    //When date and month picker are side-by-side and "Go to today" button is visible no additional margin is needed
+    .wrap.goTodaySpacing{
+      margin-bottom: 0px;
```

Comment:
The 'Go to today' button is position absolutely, so the container has the additional margin so the layout around the button does not overlap it.  This was how it was done previously when the margin was always there to prevent the Go to today button from being outside the calendar component.

<img width="371" alt="screen shot 2017-08-15 at 9 19 15 am 4" src="https://user-images.githubusercontent.com/25463697/29325137-81cc9520-819b-11e7-8a71-22c252d8fa84.png">